src/Jackett.Common/Indexers/CardigannIndexer.cs

```diff
@@ -118,6 +118,14 @@ public CardigannIndexer(IIndexerConfigurationService configService, Utils.Client
                         case "text":
                             item = new StringItem { Value = Setting.Default };
                             break;
+                        case "multi-select":
+                            if (Setting.Options == null)
+                            {
+                                throw new Exception("Options must be given for the 'multi-select' type.");
+                            }
+
+                            item = new CheckboxItem(Setting.Options) { Values = Setting.Defaults };
+                            break;
                         case "select":
                             if (Setting.Options == null)
                             {
```

Comment:
I clearly have not thought this through to conclusion.
I suspect that we still need to create the .Config.variable that will contain the list of options that have been ticked in the config profile.
I think the code block may be within 
https://github.com/Jackett/Jackett/blob/d2895ea4a5370e66d0db129b9e6b5f488b1d0fce/src/Jackett.Common/Indexers/CardigannIndexer.cs#L205-L229

then in theory, I can use something in yaml like 
`- path "search/?search={{ .Keywords }}{{ if .Config.codec }}&tags={{ join .Config.codec \";\"}}{{end}}"
or similar
am I right?

Reply:
Hmm. Possibly, yeah. I'll take a look at it. I realized myself earlier that adding multi-select to the cardigann indexer only solves half the problem and is rather useless if its value(s) can't be used elsewhere in the YAML.